index.html

```diff
@@ -878,8 +883,14 @@ <h3>
           <li>Initialize |buttons| to be an empty [=list=].
           </li>
           <li>[=list/For each=] |buttonIndex:long| of [=the range=] from 0 to
-          |buttonsSize| − 1, [=list/append=] a [=new=] {{GamepadButton}} to
-          |buttons|.
+          |buttonsSize| − 1:
+            <ol>
+              <li>Let |button:GamepadButton| be a [=new=] {{GamepadButton}} with its
+              {{GamepadButton/type}} attribute initialized to |type|.
```

Comment:
We're initializing button.type but the remaining GamepadButton attributes are left uninitialized. Let's specify that button.value is initialized to zero and button.pressed and button.touched are initialized to false.

Reply:
@nondebug  Thanks Matt for the review. Just updated the PR.